api/dash0monitoring/v1alpha1/dash0monitoring_types.go

```diff
@@ -169,18 +151,18 @@ func (d *Dash0Monitoring) SetAvailableConditionToUnknown() {
 	meta.SetStatusCondition(
 		&d.Status.Conditions,
 		metav1.Condition{
-			Type:    string(ConditionTypeAvailable),
+			Type:    string(common.ConditionTypeAvailable),
 			Status:  metav1.ConditionUnknown,
 			Reason:  "ReconcileStarted",
-			Message: "Dash0 has started resource reconciliation.",
+			Message: "Dash0 has started resource reconciliation.", // TODO
 		})
 	meta.SetStatusCondition(
 		&d.Status.Conditions,
 		metav1.Condition{
-			Type:    string(ConditionTypeDegraded),
+			Type:    string(common.ConditionTypeDegraded),
 			Status:  metav1.ConditionTrue,
 			Reason:  "ReconcileStarted",
-			Message: "Dash0 is still starting.",
+			Message: "Dash0 is still starting.", // TODO
```

Comment:
nit: please do not add TODOs to the code base in commits. If they are a reminder for yourself to revisit something, I would prefer you leave those in your local diff. There are already a couple of TODOs in the code base from earlier PRs. 

In particular, TODOs with no explanation are not helpful at all.